docs/src/man/examples.md

```diff
@@ -3,72 +3,37 @@
-- [ExtractMacro.jl](https://carlobaldassi.github.io/ExtractMacro.jl/stable/)
-- [EzXML.jl](https://juliaio.github.io/EzXML.jl/stable/)
-- [FourierFlows.jl](https://fourierflows.github.io/FourierFlowsDocumentation/stable/)
-- [Gadfly.jl](https://gadflyjl.org/stable/)
-- [GeophysicalFlows.jl](https://fourierflows.github.io/GeophysicalFlowsDocumentation/stable/)
-- [GeoStats.jl](https://juliaearth.github.io/GeoStatsDocs/stable/)
-- [Highlights.jl](https://highlights.juliadocs.org/stable/)
-- [HypothesisTests.jl](https://juliastats.org/HypothesisTests.jl/stable/)
-- [InvertibleNetworks.jl](https://slimgroup.github.io/InvertibleNetworks.jl/stable/)
-- [JUDI.jl](https://slimgroup.github.io/JUDI.jl/stable/)
-- [Luxor.jl](https://juliagraphics.github.io/Luxor.jl/stable/)
-- [MergedMethods.jl](https://michaelhatherly.github.io/MergedMethods.jl/stable/)
-- [Mimi.jl](https://www.mimiframework.org/Mimi.jl/stable/)
-- [NumericSuffixes.jl](https://michaelhatherly.github.io/NumericSuffixes.jl/stable/)
-- [NLOptControl.jl](https://huckl3b3rry87.github.io/MPCDocs.jl/stable/)
-- [Oceananigans.jl](https://clima.github.io/OceananigansDocumentation/stable/))
-- [OhMyREPL.jl](https://github.com/KristofferC/OhMyREPL.jl)
-- [OnlineStats.jl](https://joshday.github.io/OnlineStats.jl/stable/)
-- [PhotoAcoustic.jl](https://slimgroup.github.io/PhotoAcoustic.jl/stable/)
-- [Pigeons.jl](https://pigeons.run/dev/)
-- [POMDPs.jl](https://juliapomdp.github.io/POMDPs.jl/stable/)
-- [PhyloNetworks.jl](https://crsl4.github.io/PhyloNetworks.jl/stable/)
-- [PrivateModules.jl](https://michaelhatherly.github.io/PrivateModules.jl/stable/)
-- [Query.jl](https://www.queryverse.org/Query.jl/stable/)
-- [QuestDBClient.jl](https://ochibobo.github.io/QuestDBClient.jl/dev/)
-- [SimGBS.jl](https://anshess.github.io/SimGBS.jl/stable/)
-- [SlimOptim.jl](https://slimgroup.github.io/SlimOptim.jl/stable/)
-- [SymbolicRegression.jl](https://astroautomata.com/SymbolicRegression.jl/stable/)
-- [TaylorSeries.jl](https://juliadiff.org/TaylorSeries.jl/stable/)
-- [Weave.jl](https://weavejl.mpastell.com/stable/)
-- [WorldDynamics.jl](https://worlddynamics.github.io/WorldDynamics.jl/stable/)
-
-## Documentation repositories
+Nowadays most of Julia packages use Documenter.jl to build their documentation.
+The following sites are based on Documenter.jl and so should give a good overview of what this package is currently able to do.
+Please open a PR if the following description is not up-to-date.
+
+- [Julia](https://docs.julialang.org/en/v1/)
+  - The Julia official documentation.
+- [Documenter.jl](https://documenter.juliadocs.org/dev/)
+  - The documentation of Documenter.jl is built with Documenter.jl of course.
+  - The source of the docs might be complicated for beginngers, but still a good reference.
+- [DrWatson.jl](https://juliadynamics.github.io/DrWatson.jl/dev/)
+  - DrWatson.jl has nice CSS customization which can be good reference.
```

Comment:
I'd keep this list in a more compressed format, focusing on what is special in their setup. Use the format `- [name of package](link) – (en-dash) single abstract that explains what is special about the `Documenter` setup of this particular package`.

```
- [Julia](https://docs.julialang.org/en/v1/) — the official Julia documentation
- [Documenter.jl](https://documenter.juliadocs.org/dev/) – the `Documenter` documentation serves as its own showcase
- [DrWatson.jl](https://juliadynamics.github.io/DrWatson.jl/dev/) – use of custom CSS
...
```